src/Markdig.Wpf/Themes/generic.xaml

```diff
@@ -61,4 +62,16 @@
     <Setter Property="Stretch" Value="Fill" />
     <Setter Property="Stroke" Value="Black" />
   </Style>
+    
+  <!-- MarkdownViewer Template-->  
+  <Style TargetType="markdig:MarkdownViewer">
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate TargetType="markdig:MarkdownViewer">
+          <FlowDocumentScrollViewer Document="{TemplateBinding Document}"
+                                    ScrollViewer.VerticalScrollBarVisibility="Auto"/>
```

Comment:
Yes that would indeed be a better way to do it. It's not something I've ever tried I don't think, but I think it should be possible.